controllers/serviceController_int_test.go

```diff
@@ -127,18 +176,63 @@ var _ = Describe("Creating ingress objects with the ingress generator", func() {
 			Expect(expectedIngress.Items[0].Spec.Rules[0].HTTP.Paths[0].Backend.Service.Name).Should(Equal("nexus"))
 			Expect(expectedIngress.Items[0].Spec.Rules[0].HTTP.Paths[0].Backend.Service.Port.Number).Should(Equal(int32(8082)))
 
-			_, ok := expectedIngress.Items[0].Annotations[ingressRewriteTargetAnnotation]
+			_, ok = expectedIngress.Items[0].Annotations[ingressRewriteTargetAnnotation]
 			Expect(ok).Should(BeFalse())
-
-			Expect(expectedIngress.Items[1].Namespace).Should(Equal(myNamespace))
-			Expect(expectedIngress.Items[1].Name).Should(Equal("nexus-docker-registry"))
-			Expect(*expectedIngress.Items[1].Spec.IngressClassName).Should(Equal(myIngressClassName))
-			Expect(expectedIngress.Items[1].Spec.Rules[0].HTTP.Paths[0].Path).Should(Equal("/v2"))
-			Expect(*expectedIngress.Items[1].Spec.Rules[0].HTTP.Paths[0].PathType).Should(Equal(networking.PathTypePrefix))
-			Expect(expectedIngress.Items[1].Spec.Rules[0].HTTP.Paths[0].Backend.Service.Name).Should(Equal("nexus"))
-			Expect(expectedIngress.Items[1].Spec.Rules[0].HTTP.Paths[0].Backend.Service.Port.Number).Should(Equal(int32(8082)))
-			Expect(expectedIngress.Items[1].Annotations[ingressRewriteTargetAnnotation]).Should(Equal("/nexus/repository/docker-registry/v2"))
 		})
+
+		//It("Should create ingress object for multiple ces services", func() {
```

Comment:
likewise: deadcode

Reply:
I commented them out for faster feedback. I commented them back in.